README.md

```diff
@@ -87,6 +87,8 @@ This RFC makes no requirement as to how the transition should happen, but here a
 
 - The existing categorization of packages gets lost. Counter-arguments:
   - It was never that useful to begin with
+    - The categorization was always incomplete, because packages defined in the language package sets often don't get their own categorized file path.
+    - It was an inconvenient user interface, requiring a checkout or browsing through GitHub
```

Comment:
```suggestion
    - It was an inconvenient user interface, requiring a checkout or browsing through GitHub
    - Many packages fit multiple categories, leading to multiple locations to search through instead of one
```